mesh/storage.c

```diff
@@ -515,9 +470,13 @@ bool storage_load_nodes(const char *dir_name)
 
 bool storage_create_node_config(struct mesh_node *node, void *data)
 {
+	const uint8_t NUM_OF_SEP_IN_UUID_STR = 4;
+	const uint8_t uuid_str_len =
+		(2 * UUID_LEN) + NUM_OF_SEP_IN_UUID_STR + 1;
```

Comment:
`#define` this along with `UUID_LEN`, as:
`#define UUID_STR_LEN (2 * (UUID_LEN) + 4)` (note omitted +1 for NULL termination)

Reply:
Done